src/kdalgorithms.h

```diff
@@ -25,12 +25,14 @@
 #include "kdalgorithms_bits/transform.h"
 #include "kdalgorithms_bits/zip.h"
 #include <algorithm>
+#include <map>
 #include <numeric>
 #include <type_traits>
 #include <utility>
 #include <vector>
 
 #if __cplusplus >= 201703L
+#include <map>
```

Comment:
Already included in  line 28

Reply:
ahh indeed, I failed to do a proper self-review before pushing, sorry but thanks.